lib/postgres.dart

```diff
@@ -17,6 +17,8 @@ export 'src/exceptions.dart';
 export 'src/pool/pool_api.dart';
 export 'src/replication.dart';
 export 'src/types.dart';
+export 'src/types/geo_types.dart';
+export 'src/types/range_types.dart' hide Range, DiscreteRange, ContinuousRange;
```

Comment:
Could these types be private types, e.g. `_Range`, `_DiscreteRange` and `_ContinuousRange`, so that it is explicit in the code that they are not intended to be public.

On the other hand `Range` has useful public methods, maybe it should be also public? I'm not sure why would we hide these?

Reply:
removed `hide` expression